portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path

```diff
@@ -60,7 +60,7 @@
 </tr>
 <tr>
 	<td>DATA_SET_VIEW_ENTRY_NAME</td>
-	<td>//li[contains(@class,'list-group-item')]//div[contains(@class,'justify-content-center')]//p[(@class="list-group-title") and contains (.,'${entryName}')]</td>
+	<td>//li[contains(@class,'list-group-item')]//div[contains(@class,'justify-content-center')]//*[(@class="table-list-title") and contains (.,'${entryName}')]</td>
```

Comment:
Could you check if the path FrontendDataSetAdmin#TABLE_CELL is working? Because the changes on this PR may cause a path change for TABLE_CELL too.